test/e2e/service.go

```diff
@@ -859,13 +869,19 @@ spec:
 			framework.Failf("Unable to write CRD config to disk: %v", err)
 		}
 		framework.Logf("Create the Load Balancer configuration")
-		framework.RunKubectlOrDie("default", "create", "-f", loadBalancerYaml)
+		e2ekubectl.RunKubectlOrDie("default", "create", "-f", loadBalancerYaml)
 
 	})
 
 	ginkgo.AfterEach(func() {
+		if ginkgo.CurrentGinkgoTestDescription().Failed {
+			dumpBGPInfo(reportPath, ginkgo.CurrentSpecReport().LeafNodeText, f)
+			k8sReporter := InitReporter(framework.TestContext.KubeConfig, reportPath,
+				[]string{metallbNamespace, ovnNamespace, namespaceName})
```

Comment:
Why do we need ovnNamespace? Isn't that already collected?

Reply:
correct, pod logs is collected with kind logs collection. but we may need to collect other things like crs, node dumps, pod specs, recent pod logs etc. dumpInfo can be used for that.